src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -45,7 +50,18 @@ export function fetchPaymentInformation(recordEvent = recordAnalyticsEvent) {
         response,
       });
     } else {
-      recordEvent({ event: 'profile-get-direct-deposit-retrieved' });
+      recordEvent({
+        event: 'profile-get-direct-deposit-retrieved',
+        // The API might report an empty payment address for some folks who are
+        // already enrolled in direct deposit. But we want to make sure we
+        // always treat those who are signed up as being eligible. Therefore
+        // we'll check to see if they either have a payment address _or_ are
+        // already signed up for direct deposit here:
+        'direct-deposit-setup-eligible':
```

Comment:
Adding these `direct-deposit-setup-eligible ` and `direct-deposit-setup-complete` keys is the meat of this PR.